indico/modules/events/abstracts/blueprint.py

```diff
@@ -39,6 +40,8 @@
 # Display pages
 _bp.add_url_rule('/abstracts/<int:abstract_id>/', 'display_abstract', RHDisplayAbstract)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
+_bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
```

Comment:
We don't need one but it's much prettier to have one (we do the same thing for other download endpoints too)